### ISSUE TITLE
Node class idea for if/else statement

### ISSUE THREAD

**Issue by askiefer** [MinesJA/grid#5]
https://github.com/MinesJA/grid/blob/c1651d9003bfd26d0811dfb90d0ff60a84077c5a/grid/commands/commands.py#L12

So maybe for this you can just define an `add_sibling()` method on each of the message types (Tell, Ask, Response) and then call that directly, something like: 

```
class Animal:
    pass

class Dog(Animal):
    def talk(self):
        print("Woof")


class Cat(Animal):
    def talk(self):
        print("Meow")


def some_function(some_animal: Animal):
    some_animal.talk()


if __name__ == "__main__":
    cat = Cat()
    dog = Dog()

    some_function(cat)
    some_function(dog)
```

Obviously passing in the envelop, and node as well. This may grossly oversimplify things lol, and if what you have works I think it's fine too!

**Comment by MinesJA** [comment]
Yeah, this is exactly what I want to do but I keep getting stuck with a circular dependency error which makes me think I'm missing something in the design. Currently message execution can propogate other messages and Python is not liking that.

So here's a good example of what I'm trying to do: https://replit.com/join/ecizphxhfa-minesja89

This is probably an easier fix than I realize and I'm just missing something....
